src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,9 @@
+package wolox.training.repositories;
+
+import org.springframework.stereotype.Repository;
+import wolox.training.models.Book;
+
+public interface BookRepository extends Repository {
```

Comment:
You could extend from CrudRepository<Book, Integer>, which returns Optional<Book> in the methods :)